### PR TITLE
Upload cmd - redesign vcs props

### PR DESCRIPTION
**Pull request by Or-Gabay**

### CODE REVIEW

**Comment by Or-Gabay** [comment]
Related PR: https://github.com/jfrog/jfrog-client-go/pull/108

File: artifactory/commands/generic/upload.go [hunk 1]
```diff
@@ -70,6 +70,7 @@ func (uc *UploadCommand) upload() error {
 	// Initialize Progress bar, set logger to a log file
 	var err error
 	var progressBar ioUtils.Progress
+	AddVcsProps := false
```
Comment (eyalbe4) [inline artifactory/commands/generic/upload.go]: AddVcsProps --> addVcsProps

File: artifactory_test.go [hunk 2]
```diff
@@ -4015,3 +4017,37 @@ func TestVcsProps(t *testing.T) {
 	tests.RenamePath(newinnerDotGitPath, filepath.FromSlash((oldinnerDotGitPath + "/gitdata")), t)
 	cleanArtifactoryTest()
 }
+
+// Rename the ".git" dir to it origin name "gitdata" in vcs testdata dir.
+func cleanVcsTestDir(t *testing.T) {
```
Comment (eyalbe4) [inline artifactory_test.go]: This function seems to be changing the files which are part of *testdata*, which is part of the project sources.
The tests are not supposed to change the sources.

Comment (Or-Gabay) [reply artifactory_test.go]: Removed.

**Pull request merged**